api/base/renderers.py

```diff
@@ -31,3 +31,14 @@ def get_context(self, *args, **kwargs):
         for form in unwanted_forms:
             del context[form]
         return context
+
+
+class PlainTextRenderer(StaticHTMLRenderer):
+    """
+    Renders plain text.
+
+    Inherits from StaticHTMLRenderer for error handling.
+    """
+
+    media_type = 'text/markdown'
+    format = '.txt'
```

Comment:
Oh okay! It's hard to tell the difference because the route is only rendering wiki content and not downloading it.